agent_node:
  ros__parameters:
    name: 
    algo_idx: 
    scan_topic: scan
    drive_topic: drive
